packages/hop-node/src/db/TransferRootsDb.ts

```diff
@@ -574,6 +575,11 @@ class TransferRootsDb extends BaseDb {
           Date.now()
       }
 
+      let settleAttemptTimestampOk = true
+      if (item.settleAttemptedAt) {
+        settleAttemptTimestampOk = item.settleAttemptedAt + OneHourMs < Date.now()
```

Comment:
We should use `TxRetryDelayMs` instead of `OneHourMs`. We should have probably been doing that this whole time.

Reply:
Updated

Resolved with https://github.com/hop-protocol/hop/pull/386/commits/2486709139caafa02c59530489d107b6965ccba2